content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses_locations.rst

```diff
@@ -8,88 +8,110 @@ Terminology
 Warehouse
 ---------
 
-In Odoo, a *warehouse* is the actual building/place in which a company's items are stocked. Multiple
-warehouses can be set up in Odoo and the user can create moves between warehouses.
+In Odoo *Inventory*, a *warehouse* is a physical building or space where items are stored. By
+default, each Odoo database has one warehouse already pre-configured, with the address set as
+the company's address.
```

Comment:
```suggestion
default, each Odoo database has one warehouse already pre-configured, with the address set as the
company's address.
```
early line break on 12 (double check with make review, but moving "the" from 13 to 12 should fix it)

Reply:
Nice catch - Suggestion implemented ✅